apps/chat_with_github/app.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import logging
+import base64
+from pathlib import Path
+
+import streamlit as st
+from dotenv import load_dotenv
+from camel.logger import set_log_level
+from camel.toolkits import MCPToolkit
+from camel.agents import ChatAgent
+from camel.models import ModelFactory
+from camel.types import ModelPlatformType, ModelType
+
+# Silence noisy asyncio cancellation messages
+logging.getLogger("asyncio").setLevel(logging.CRITICAL)
+
+# ——— Page Config ———
+st.set_page_config(page_title="Chat with your Repo", layout="wide")
+
+# ——— Load assets & logos ———
+ROOT = Path(__file__).resolve().parents[2]
+ASSETS = ROOT / "assets"
```

Comment:
Should be like this?
```suggestion
ROOT = Path(__file__).resolve().parent
```